azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -186,12 +191,24 @@ internal class CallingSDKWrapper(
                             arrayOf(videoStream.native as NativeLocalVideoStream)
                         videoOptions = VideoOptions(localVideoStreams)
                     }
-                    joinCall(agent, audioOptions, videoOptions, callLocator)
+                    callLocator?.let {
+                        joinCall(agent, audioOptions, videoOptions, callLocator)
+                        return@whenComplete
+                    }
+                    callConfig.participants?.let {
+                        startCall(agent, audioOptions, videoOptions, it)
+                    }
                 }.exceptionally { error ->
                     onJoinCallFailed(startCallCompletableFuture, error)
                 }
             } else {
-                joinCall(agent, audioOptions, videoOptions, callLocator)
+                callLocator?.let {
+                    joinCall(agent, audioOptions, videoOptions, callLocator)
```

Comment:
Let's use explicit joinCall(agent, audioOptions, NULL, callLocator) as videoOptions is always null here